apps/src/javalab/JavalabSettings.jsx

```diff
@@ -69,8 +69,10 @@ export class JavalabSettings extends Component {
       <div style={styles.main}>
         <JavalabButton
           icon={<FontAwesome icon="cog" />}
+          text={'Settings'}
```

Comment:
nit: move this string to i18n. there's likely already a "Settings" string in common/en_us.json that you could use